packages/contracts-bedrock/contracts/test/ResourceMetering.t.sol

```diff
@@ -134,6 +141,23 @@ contract ResourceMetering_Test is Test {
         assertEq(postBaseFee, 2125000000);
     }
 
+    function test_meter_baseFeeMaxChangeDenominator_reverts() external {
```

Comment:
This part of the name should be descriptive:

```suggestion
    function test_meter_denominatorEq1_reverts() external {
```

Reply:
This test is a little funny since it's testing behaviour that is only  exposed by the `MeterUser` contract, and shouldn't technically be reachable in the base implementation. 

So, I think it should also have a comment in natspec to explain what it's doing.